src/Module.ts

```diff
@@ -418,6 +447,25 @@ export default class Module {
 		}
 	}
 
+	getDynamicImportExpressions (): (string | Node)[] {
+		return this.dynamicImports.map(node => {
+			let dynamicImportExpression: string | Node;
+			const importArgument = node.parent.arguments[0];
+			if (importArgument.type === NodeType.TemplateLiteral) {
```

Comment:
If you use type guards such as `isLiteral` (which is exported by Literal.js), all of the type casts become redundant and it becomes even more readable.

Reply:
Also, what happens if this check is not satisfied? It seems to me that in this case, you would return undefined instead of e.g. `importArgument`. Is this intended?

Reply:
Yes an undefined return is permitted, which indicates a no-op.